custom_components/bosch_alarm/translations/en.json

```diff
@@ -13,8 +14,10 @@
                 "data": {
                     "host": "Host",
                     "port": "Port",
-                    "password": "Automation Passcode \\ RSC+ Passcode"
-                }
+                    "password": "Automation code",
+                    "installer_code": "Installer code"
+                },
+                "description": "For Solution and AMAX panels, provide both your automation code and installer code.\nFor other panels, provide just your automation code.\nNote that your automation code needs to be at least 10 characters long, otherwise you won't be able to connect to your panel. \nThe automation code can be set in A-Link or RPS.\nYour installer code should have been provided by whoever set up your panel."
```

Comment:
I think we can use [data_description](https://developers.home-assistant.io/docs/data_entry_flow_index#show-form) to put the instructions next to the field they relate to. 

Also, it would sound a bit more professional to avoid the second person.

How about

"password": The Automation Code is required to access all panels using the Mode 2 protocol. It can be configured via the A-Link Plus or the Remote Programming Software (RPS). Typically, it is a 10 character string.
"installer_code": (only required for Solution and AMAX panels) The Installer Code is used to access the configuration interface. Typically, it is a 4 digit numeric pin.

Reply:
The reason for specifying 10 characters like this is because you can set it to a smaller string in RSC+ but then things just don't work

Reply:
Sure, we can make the wording stronger, i.e. "It must be ...". Do we know if it has to be exactly 10, or at least 10?

Reply:
At least iirc